mmpose/core/evaluation/top_down_eval.py

```diff
@@ -319,6 +388,14 @@ def keypoints_from_heatmaps(heatmaps,
         kernel (int): Gaussian kernel size (K) for modulation, which should
             match the heatmap gaussian sigma when training.
             K=17 for sigma=3 and k=11 for sigma=2.
+        keypoint_pose_distance (float): Keypoint pose distance for UDP.
```

Comment:
I'm trying to understand the meaning of `keypoint pose distance`. Is it the distance of keypoint to pose?
If not, does it have more precise/descriptive name?

Reply:
this is first proposed in google paper  ''Towards Accurate Multi-person Pose Estimation in the Wild'' where use 'radius' as denotation, and we explain this as the valid distance from the annotated keypoint coordiantes，and the pixels in the heatmap within this distance should be classify as positive in the heatmap.  'keypoint_pose_distance' is not the name i design and is inhering from other repo~ may be the original 'radius' is more reasonable?
![image](https://user-images.githubusercontent.com/31001081/101570751-07d2a600-3a12-11eb-982c-258d4483b407.png)

Reply:
Thanks for the clear explanation. How about `valid_radius`? And in the docstring, give it concreate descriptions like `The predicted positions within a disk of radius R from the location of a keypoint is considered as correct prediction to that keypoint`.

Reply:
'valid_radius' is better for understanding.  i prefer to explain this as 'The radius of the positive area in classification heatmap.'

Reply:
Yeah that's better